doc/nrf/protocols/thread/sed_ssed.rst

```diff
@@ -55,6 +55,81 @@ The following Kconfig options affect the size of the receive window, and thus al
 Additionally, you must configure the ``period`` CLI parameter to enable CSL.
 For more information on using the CLI to configure parameters for CSL, see the ``csl`` command in the OpenThread `CLI reference <OpenThread CLI Reference - csl command_>`_.
 
+.. _thread_ssed_fine_tuning:
+
+SSED fine tuning
+================
+
+SSED devices are synchronized with the parent router, and they are scheduled to work at specific time.
+During the attach process, the child SSED device receives these parameters from the parent router.
+The configuration of the SSED child device must be applied to the parent router and it impacts on child SSED power consumption.
+However, the SSED parameters like `CSL Window edges` configuration which are applied to each SSED device independently, must be tuned carefully to ensure reliable reception while minimizing power consumption.
+The following two parameters and Kconfig options are used to tune the SSED device on the parent router:
+
+* CSL uncertainty - :kconfig:option:`CONFIG_OPENTHREAD_PLATFORM_CSL_UNCERT`
+* CSL accuracy - :kconfig:option:`CONFIG_NRF5_DELAY_TRX_ACC` (or :kconfig:option:`CONFIG_IEEE802154_NRF5_DELAY_TRX_ACC` while using :ref:`OpenThread with Zephyr L2 Networking Layer <ug_thread_configuring_basic>`)
+
+The following figure shows the time diagram of an SSED device working.
+
+.. figure:: overview/images/thread_ssed.png
+   :alt: SSED time diagram
+
+   SSED time diagram
+
+The ``CSL uncertainty`` represents the worst-case deviation between the actual arrival time of a transmission and the expected transmission time, as calculated relative to the local radio clock, regardless of elapsed time.
+The value of ``CSL uncertainty`` is specified in units of 10 microseconds.
+A smaller value of ``CSL uncertainty`` reduces the device's power consumption, but it also increases the risk of missing transmissions from the parent router.
+
+In theory, the ``CSL uncertainty`` should be set to ``1`` (which represents 10 microseconds) to ensure that the device is always ready to receive data.
+However, in practice, this value is higher due to the following reasons:
+
+* For some reason, the crystal oscillator (for example, 32.768 kHz) without additional clock multiplier may be used, and this limits the resolution of time ticks to ~30.5 microseconds.
+  This assumes ±31 us of uncertainty, however it can be reduced to ±16 if the platform code is carefuly designed.
+* If the OpenThread implementation of CSL scheduling is used, then the uncertinity may be higher due to the known `OpenThread CSL issue`_.
+* MCU interrupt handling may introduce additional uncertainty of a few microseconds.
+* Bugs in the network stack may cause some suprious timing deviations, and instead of fixing this bugs, vendor could decide to increase the uncertainty to compensate for this.
```

Comment:
Do we need to say Bugs? Maybe just say that Network stack may introduce some spurious timing deviations etc.